mw_validate_json.go

```diff
@@ -44,10 +44,6 @@ func (k *ValidateJSON) ProcessRequest(w http.ResponseWriter, r *http.Request, _
 		return errors.New("no schemas to validate against"), http.StatusInternalServerError
 	}
 
-	if vPathMeta.SchemaVersion != "" && vPathMeta.SchemaVersion != "draft-v4" {
```

Comment:
Why you removed it?

Reply:
because the schema itself supports versioning. unless I misunderstood? http://json-schema.org/latest/json-schema-core.html#rfc.section.7